blivet/devices/partition.py

```diff
@@ -50,6 +50,8 @@
 # in case the default partition size doesn't fit
 FALLBACK_DEFAULT_PART_SIZE = Size("256MiB")
 
+EXTENDED_METADATA_SIZE = Size("1 MiB")
+
```

Comment:
So using grainSize instead of 1 MiB should be enough here? I wasn't able to find any reliable source with information about the actual metadata size for extended partitions and 1 MiB is the space between start of extended partition and start of first logical one. (And I didn't realize this is the same as the grain size, so it makes sense to use grain size instead of some "random" number.)